oam/install_scripts/mariadb-columnstore-start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This script allows to gracefully start MCS
+
+/bin/systemctl start mcs-loadbrm
+/bin/systemctl start mcs-workernode
+/bin/systemctl start mcs-controllernode
+/bin/systemctl start mcs-exemgr
+/bin/systemctl start mcs-storagemanager
```

Comment:
I'm unclear on the difference between using the 'umbrella' systemd service vs this script.  If this script is synchronous, that is, one start cmd finishes before another one can start, and not firing off all of these start cmds in parallel, then storagemanager needs to start first because all file IO goes through it.  Also primproc before exemgr.  Otherwise, I think the ordering is OK.

Reply:
Despite the fact we can rely on systemd dependenices here I'll change the order to make it explicit for an external viewer.